packages/salesforcedx-vscode-lwc/src/testSupport/workspace/getCliArgsFromJestArgs.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { TestRunType } from '../testRunner/testRunner';
+import { workspaceService } from './workspaceService';
+
+/**
+ * Returns workspace specific jest args from CLI arguments and test run type
+ * @param jestArgs jest args
+ * @param testRunType test run type
+ */
+export function getCliArgsFromJestArgs(
+  jestArgs: string[],
+  testRunType: TestRunType
+) {
+  const workspaceType = workspaceService.getCurrentWorkspaceType();
+  if (testRunType === TestRunType.DEBUG) {
+    if (
+      workspaceService.isSFDXWorkspace(workspaceType) ||
+      workspaceService.isCoreWorkspace(workspaceType)
+    ) {
+      return ['--debug', '--', ...jestArgs];
```

Comment:
I don't think the order of the arguments matters. If that is true, you can restructure the code so you create an array of options depending on the workspace type and add the debug argument at the end if needed. That will remove a lot of the code duplication.

Reply:
This is actually intentional to account for the test runner differences. The order matters here and I chose to explicitly declare the parameters since the test runners are quite different (esp. `create-lwc-app` test runner)

Reply:
That's fine if the order matters, I think there are still ways in which you can rewrite the code so that there's no duplication of conditions if you want to enable/disable the debug flag.

Reply:
Updated the function and simplified the current implementation. For reference [this](https://github.com/forcedotcom/salesforcedx-vscode/blob/4b194842168f5189ba26529d2fe340cbff64aac9/packages/salesforcedx-vscode-lwc/src/testSupport/workspace/getCliArgsFromJestArgs.ts) is the original commit in case we need to do it differently in the future.